BenefitsAllocationUpload/Controllers/ExcludedObjectController.cs

```diff
@@ -0,0 +1,259 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Web;
+using System.Web.Http;
+using System.Web.Mvc;
+using BenefitsAllocation.Core.Domain;
+using BenefitsAllocationUpload.Models;
+using UCDArch.Core.PersistanceSupport;
+using ExcludedObject = BenefitsAllocation.Core.Domain.ExcludedObject;
+
+namespace BenefitsAllocationUpload.Controllers
+{
+    public class ExcludedObjectController : ApplicationController
+    {
+        private readonly IRepositoryWithTypedId<ExcludedObject, ExcludedObjectId> _excludedObjectRepository;
+
+        public ExcludedObjectController(
+            IRepositoryWithTypedId<ExcludedObject, ExcludedObjectId> excludedObjectRepository)
+        {
+            _excludedObjectRepository = excludedObjectRepository;
+        }
+
+        //
+        // GET: /ExcludedObject/
+
+        /// <summary>
+        /// Return a list of reimbursable excluded objects for the logged in user's Org. 
+        /// </summary>
+        /// <returns></returns>
+        public ActionResult Index()
+        {
+            var orgId = GetOrgIdForCurrentUser();
+            if (!String.IsNullOrEmpty(orgId))
+            {
+                var results =
+                    _excludedObjectRepository.Queryable.Where(x => x.OrgId.Equals(orgId)).ToList();
```

Comment:
i'm pretty sure this is ok in that version of EF, but not that some versions of entity framework, especially new EFCore versions, will fail here since .Equals can't be evaluated on the client side.  It'll want you to use `.Where(x => x.OrgId == orgId)` in those cases